target/linux/ar71xx/base-files/etc/diag.sh

```diff
@@ -30,6 +30,9 @@ get_status_led() {
 	xd3200)
 		status_led="$board:green:system"
 		;;
+	ap531b0)
```

Comment:
Please, follow the general rule and keep all boards in alphabetical order.
You have also changed file attributes (`100644 → 100755`) which is wrong - please, fix it.